classes/question/question.php

```diff
@@ -567,6 +567,10 @@ public function response_complete($responsedata) {
                 if (!empty($answer->choiceid) && isset($this->choices[$answer->choiceid]) &&
                     $this->choices[$answer->choiceid]->is_other_choice()) {
                     $answered = !empty($answer->value);
+                    $onlyspaces = preg_match("/^[\s]*$/", $answer->value);
+                    if ($onlyspaces) {
+                        $answered = false;
+                    }
                 } else {
```

Comment:
The above code could be simplified to:
`$answered = !empty($answer->value) || !preg_match("/^[\s]*$/", $answer->value);`

Reply:
Is there a case for answering "other" with no content?